DKBeatMaker/DKBeatMaker/Conductor.swift

```diff
@@ -155,11 +149,9 @@ class Conductor {
     }
 
     func startPlaying() {
-             sequence.play()
+        sequence.play()
```

Comment:
Looking at this I think I got used to have the clang formatter, so I think we should add this to the project to ensure consistency